iceoryx_hoofs/test/moduletests/test_cxx_list.cpp

```diff
@@ -294,20 +294,20 @@ TEST_F(list_test, NotFullWhenFilledWithCapacityAndEraseOneElements)
 TEST_F(list_test, NotFullWhenFilledWithCapacityAndEraseOneAndReinsertElements)
 {
     ::testing::Test::RecordProperty("TEST_ID", "c7e70ac7-e476-43aa-976c-1ac78513c869");
-    uint64_t i = 0U;
-    for (; i < sut.capacity(); ++i)
+    int64_t counter = 0U;
```

Comment:
I think `counter` should be unsigned to avoid the casts.

Has multiple occurrences in the following tests.